### PR TITLE
Trestle release

### PR DESCRIPTION
**Pull request by butler54**
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [x] All commits are signed-off.

## Summary
- Fixes critical bugs in the trestle author command
- Additional feature of `--header-merge` flag for trestle author *-generate.
## Key links:

- [Sonar coverage](https://sonarcloud.io/dashboard?id=compliance-trestle)

### CODE REVIEW

**Comment by butler54** [comment]
Do not merge until after #858 has been merged.

**Review by fsuits — APPROVED** [review]
LGTM

**Review by enikonovad — APPROVED** [review]
LGTM

**Pull request merged**
